Readme.md

```diff
@@ -12,51 +10,52 @@ for a guide to upgrading from 1.x to 2.x.
 
 -----
 
-### Canvas graphics API backed by Cairo
 [![Build Status](https://travis-ci.org/Automattic/node-canvas.svg?branch=master)](https://travis-ci.org/Automattic/node-canvas)
 [![NPM version](https://badge.fury.io/js/canvas.svg)](http://badge.fury.io/js/canvas)
 
-  node-canvas is a [Cairo](http://cairographics.org/) backed Canvas implementation for [NodeJS](http://nodejs.org).
-
-## Authors
-
-  - TJ Holowaychuk ([tj](http://github.com/tj))
-  - Nathan Rajlich ([TooTallNate](http://github.com/TooTallNate))
-  - Rod Vagg ([rvagg](http://github.com/rvagg))
-  - Juriy Zaytsev ([kangax](http://github.com/kangax))
+node-canvas is a [Cairo](http://cairographics.org/)-backed Canvas implementation
+for [Node.js](http://nodejs.org).
```

Comment:
I _personally_ prefer having every paragraph on one line, and letting the editor wrap, simply because the diffs can get very weird when adding a small thing to one line that overflows each subsequent line. That said, I don't have a very strong opinion on this so I'll let you make the call ☺️

Reply:
Me too! I just wanted it one way or another. I'll remove the line breaks.